### DISCUSSION TITLE
Problem in Hill model based DDI

### DISCUSSION THREAD

**Discussion by HUA0827** [Open-Systems-Pharmacology/Forum#1300] [Questions & Problems] [answered]
Hi all,
I had a problem in developing a PBPK model to investigate the DDI between a CYP3A substrate and inhibitors.
In my simulation, metabolism inhibition could be observed by first order or MM process but not hill model for competitive inhibition. 
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/17555865/189685678-c4938693-cb3b-43c8-8b4a-9155a0ce82e7.png">

As mechanistic inhibition could be simulated, we deduced that the problem might be related to the km,app. This deduction has been validated by sending the simulation to MoBi. In MoBi, km,app was not estimated when hill model was used. 
Can I solve this problem in PK-sim? Or MoBi is required to fix it?
I'm look forward to your response. Thanks!

**Comment by StephanSchaller** [comment]
If this is a bug (or a yet to be implemented feature), you can only solve this in MoBi as a user, and it will have to be fixed by the software developers for PK-Sim.

@sfrechen , is this known?

**Comment by msevestre** [comment]
@HUA0827 Which version of the software are you using??

**Reply by HUA0827** [reply]
Version 11

**Comment by Yuri05** [comment, marked as answer]
Known issue - DDI for the Hill kinetic is not implemented yet. https://github.com/Open-Systems-Pharmacology/Forum/issues/351
We should mention it in the docu (https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1283)

**Comment by HUA0827** [comment]
Thanks, I will tried it in MoBi.
